### PR TITLE
Feature/create team

### PR DESCRIPTION
**Pull request by baothaiEvizi**
implement writing 2 Test cases TGG-01, TGG-05 with method POST (Create a team).

### CODE REVIEW

File: helper/constants.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "Training Selenium, Postman",
+    "Using Selenium",
+    "Using Postman"
+];
+
+export const baseUrl = process.env.ENDPOINT;
+export const token = process.env.TOKEN;
+
+export const randomName = names[Math.floor(Math.random() * names.length)];
+export const randomDescription = descriptions[Math.floor(Math.random() * descriptions.length)];
```
Comment (hanhEvizi) [inline helper/constants.ts:17]: Why did you add randomDescription and randomName in the constants file?

File: __tests__/teams/post_teams.spec.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        expect(response.body).toHaveProperty("id");
+        expect(response.body).toHaveProperty("is_global");
+        expect(response.body).toHaveProperty("gitguardian_url");
+        expect(response.body).toHaveProperty("name");
+        expect(response.body).toHaveProperty("description");
+        expect(typeof response.body.id).toBe("number");
+        expect(typeof response.body.is_global).toBe("boolean");
+        expect(typeof response.body.gitguardian_url).toBe("string");
+        expect(typeof response.body.name).toBe("string");
+        expect(typeof response.body.description).toBe("string");
```
Comment (hanhEvizi) [inline __tests__/teams/post_teams.spec.ts:34]: Can you refactor lines 22-34 to remove the repetition of the response.body.

File: __tests__/teams/post_teams.spec.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import request from "supertest";
+import { baseUrl, token, randomName, randomDescription } from "../../helper/constants";
+  
+describe("POST - /teams - Create a team", () => {
+    const requestBody = {
+        name: randomName,
+        description: randomDescription
+    };
```
Comment (evizi-baolonglehoang) [inline __tests__/teams/post_teams.spec.ts:9]: Should move to const file.

File: __tests__/teams/post_teams.spec.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    let team_id: number;
+
+    it("TGG-01: Verify that User can create a team", async() => {
+        const response = await request(baseUrl)
+            .post("/teams")
+            .set('Authorization', `${token}`)
+            .send(requestBody)
+
+        team_id = response.body.id;
```
Comment (evizi-baolonglehoang) [inline __tests__/teams/post_teams.spec.ts:19]: What happens if `response.body.id` return undefined or null. Should handle this to avoid the errors.

File: helper/constants.ts (hunk 1 above)
Comment (baothaiEvizi) [reply helper/constants.ts]: moved randomName and randomDescription into common.helper file

**Review by evizi-baolonglehoang — APPROVED** [review]
